libs/related-products/state/src/state.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { StoreModule } from '@ngrx/store';
+
+import { daffProductProvideMetaReducers } from '@daffodil/product/state';
+
+import {
+  DAFF_RELATED_PRODUCTS_STORE_FEATURE_KEY,
+  daffRelatedProductsDedupeMetaReducer,
+} from './reducers/public_api';
+import { daffRelatedProductsReducers } from './reducers/reducers';
+
+/**
+ * A module that provides the default reducers for the related product redux state.
+ */
+@NgModule({
+  imports: [
+    StoreModule.forFeature(DAFF_RELATED_PRODUCTS_STORE_FEATURE_KEY, daffRelatedProductsReducers),
+  ],
+  providers: [
+    ...daffProductProvideMetaReducers(daffRelatedProductsDedupeMetaReducer),
```

Comment:
I think we should continue keeping everything compatible with Pre angular 9 for now, so this would need to manually provide

Reply:
changed